app/controllers/internal/configs_controller.rb

```diff
@@ -19,6 +19,7 @@ def create
   def config_params
     allowed_params = %i[
       default_site_email social_networks_handle
+      campaign_hero_html_variant_name campaign_background_color campaign_text_color campaign_sidebar_enabled campaign_featured_tags
```

Comment:
I understand wanting to keep related attributes together, but I think at 131 characters this line is too long.